lib/sorbet-rails/rails_mixins/pluck_to_tstruct.rb

```diff
@@ -7,27 +7,37 @@ module SorbetRails::PluckToTStruct
     type_parameters(:U).
     params(
       ta_struct: ITypeAssert[T.type_parameter(:U)],
+      associations: T::Hash[Symbol, String]
```

Comment:
nit: can you add a comma
```suggestion
      associations: T::Hash[Symbol, String],
```